cli/spec/spec_helper.rb

```diff
@@ -45,22 +45,13 @@
   end
 
   config.around(:each) do |example|
-    catch :exit_with_error do
-      begin
-        example.run
-      rescue SystemExit
-        puts "Got SystemExit: #{$!.message} - Exit code: #{$!.status}"
-      end
-    end
-  end
-
-  unless ENV["DEBUG"]
-    config.before(:each) do
-      $stdout = StringIO.new
-      $stderr = StringIO.new
-    end
+    $stdout = $stderr = StringIO.new
```

Comment:
This should store the original `$stdout` / `$stderr` values and restore those in ensure instead of `$stdout = STDOUT`.

Reply:
That would probably be better than what the the original `before/after` wrapper did, yes.